src/patternfly/components/Dropdown/dropdown-menu-groups.hbs

```diff
@@ -0,0 +1,21 @@
+<ul class="pf-c-dropdown__menu{{#if dropdown-menu--modifier}} {{dropdown-menu--modifier}}{{/if}}"
+  aria-labelledby="{{id}}-button"
+  {{#unless dropdown--IsExpanded}}hidden{{/unless}}
+  {{#if dropdown-menu--attribute}}
+	  {{{dropdown-menu--attribute}}}
+  {{/if}}>
+  <li>
+    <div class="pf-c-dropdown__group-title">App</div>
+    <ul>
+      <li><a href="#" class="pf-c-dropdown__menu-item">Configuration</a></li>
```

Comment:
The other menu-items that I'm seeing in examples above, use button rather than <a></a> tag, unless it says "Link" was this purposeful?

Reply:
Yeah, you can use links or buttons in a dropdown. I can change if you you'd prefer it be a button.

Reply:
I think it said in the docs that links go off the page and buttons stay on. So i'd update it to what the other menu items use. But thats just me.

Reply:
The other menus use both. I think as long as it's clear that the component can take either, and we have examples of that, I think it's fine to use either. I'll update it to have both like the other examples though. 

<img width="1099" alt="screen shot 2019-02-20 at 1 22 04 pm" src="https://user-images.githubusercontent.com/35148959/53118438-b2fb2e00-3512-11e9-8372-b9cc24202414.png">

Reply:
okay agree with that!